forte/sci/aci.cc

```diff
@@ -227,11 +227,11 @@ void AdaptiveCI::find_q_space() {
 
     if (screen_alg == "AVERAGE") {
         if (gas_iteration_) {
-            get_gas_excited_determinants_avg(nroot_, P_evecs_, P_evals_, P_space_, F_space);
+            get_gas_excited_determinants_avg(num_ref_roots_, P_evecs_, P_evals_, P_space_, F_space);
```

Comment:
I see a lot of `nroot_ -> num_ref_roots_` changes in the aci code. I don't think they are necessary.

Reply:
I explained why they were necessary [here](https://github.com/evangelistalab/forte/pull/303#issuecomment-1228873604).